database.php

```diff
@@ -0,0 +1,7 @@
+<?php
+$connection = mysqli_connect("localhost", "root", "", "books_db");
+
+if (!$connection) {
```

Comment:
Observations: No error handling for database connection failures.
Recommendations: Implement error handling to manage connection issues gracefully.

```php
if ($connection->connect_error) {
    error_log('Connection failed: ' . $connection->connect_error);
    die("Failed to connect to database. Please try again later.");
}
```